pages/common/openssl.md

```diff
@@ -4,13 +4,29 @@
 > Some subcommands such as `req` have their own usage documentation.
 > More information: <https://www.openssl.org>.
 
-- Display help:
+- Generate a private key and encrypt the output file using AES256:
 
-`openssl help`
+`openssl genpkey -algorithm {{[rsa|ec]}} -out {{private.key}} -aes256`
```

Comment:
```suggestion
`openssl genpkey -algorithm {{rsa|ec}} -out {{private.key}} -aes256`
```
This is not the place to use this syntax. It's for the shortest shortform and most descriptive longform.